components/AddCodes/AddCodes.jsx

```diff
@@ -19,7 +19,8 @@ export default function AddCodes({ userId, releaseId, setOnCodeAdded }) {
       header: true,
       skipEmptyLines: true,
       complete: function (results) {
-        let codeArray = results.data.map((d, index) => {
+        let codeArray = []
```

Comment:
[codeArrayyyyy, codeArrayyyyy, codeArrayyyyy and save your liiiiiife](https://www.youtube.com/watch?v=_dx0qWHL7dc)